projects/observability/src/pages/apis/topology/application-flow.component.ts

```diff
@@ -15,203 +19,217 @@ import {
   template: `
     <div class="vertical-flex-layout">
       <ht-page-header></ht-page-header>
-      <ht-navigable-dashboard [navLocation]="this.location" [defaultJson]="this.defaultJson"> </ht-navigable-dashboard>
+      <ht-navigable-dashboard [navLocation]="this.location" [defaultJson]="this.getDefaultJson() | async">
```

Comment:
Ah I missed this file because the diff was too big. So the flag is meant to change the application flow page's default layout, not the component's?  Probably could use a clarifier in the name (e.g. ApplicationFlow*Page*Layout ) if we really want to go that road. Might be a good time to pull the json out of the page component as we've done on other pages too.

Reply:
The FF is for very short peroid of time, I believe. It's there just to test out the graphLayout in selected customers' env. Check for the issues or improvements and then open for all. Till then, they would be seeing the layout as it is today. Right @anandtiwary?

Reply:
As discussed, Please remove this code from here.